packages/ibm-products/src/components/ProductiveCard/ProductiveCard.tsx

```diff
@@ -5,23 +5,137 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-import React, { forwardRef } from 'react';
+import React, { PropsWithChildren, ReactNode, forwardRef } from 'react';
 import PropTypes from 'prop-types';
 import { Card } from '../Card';
 
 import { getDevtoolsProps } from '../../global/js/utils/devtools';
 import { prepareProps } from '../../global/js/utils/props-helper';
 import { pkg } from '../../settings';
+import { CarbonIconType } from '@carbon/react/icons';
 
 const componentName = 'ProductiveCard';
 
-// Default values for props
-const defaults = {
-  actionsPlacement: 'top',
+type ActionIcon = {
+  id: string;
+  icon: CarbonIconType;
+  onKeyDown?(): void;
+  onClick?(): void;
+  iconDescription: string;
+  href?: string;
 };
+type overflowAction = {
+  id?: string;
+  itemText?: string;
+  onClick?: () => void;
+  onKeydown?: () => void;
+};
+interface ProductiveCardProps extends PropsWithChildren {
+  /**
+   * Icons that are displayed on card. Refer to design documentation for implementation guidelines. Note- href will supersede onClick
+   */
+  actionIcons?: ActionIcon[];
+
+  /**
+   * Determines if the action icons are on the top or bottom of the card
+   */
+  actionsPlacement?: 'top' | 'bottom';
+  /**
+   * Content that shows in the body of the card
+   */
+  // children: PropTypes.node,
+  /**
+   * Optional user provided class
+   */
+  className?: string;
+
+  /**
+   * Designates which zones of the card are clickable. Refer to design documentation for implementation guidelines
+   */
+  clickZone?: 'one' | 'two' | 'three';
```

Comment:
Better make this as a separate type, may be can reuse in future.

Reply:
modified as advised